### ISSUE TITLE
The code result seem strange?

### ISSUE THREAD

**Issue by LYAFJJ** [yifanzhang-pro/Matrix-SSL#5]
I use the default pretrain config to pretrain the model in ImageNet and CIFAR10,and I finetune in CIFAR10.However,it can't achieve a good result.Is there any problem in the process of my attempy.

**Comment by yifanzhang-pro** [comment]
Our experiment setting is mainly focused on ImageNet pre-training and fine-tuning, I suggest you might need to adjust the hyper-parameters for better performance on CIFAR10 datasets, for example, the alignment loss ratio \gamma can be set to 0, 0.1, etc.

**Comment by LYAFJJ** [comment]
Thanks for your response.I will work on it.What's more,I find the code you use the teacher-student model and there's slight different on the structual after encoder,it refer to BYOL?Can I remove the teacher-student model and the matrix theory also has it's effect?

**Comment by yifanzhang-pro** [comment]
We use the SimSiam architecture, I think it is worth a try to remove the teacher-student model.
